astro/src/content/docs/lifecycle/migrate-users/connectors/generic-connector.mdx

```diff
@@ -98,7 +98,7 @@ To use a Generic Connector:
 
 The request to your API endpoint will be delivered as JSON:
 
-<JSON title="Example Request JSON" src="login/request.json" />
+<JSON title="Example Request JSON" src="connectors/generic-auth-request.json" />
 
 Your application will then look up the user and verify the correct credentials were provided. Then you can return the response.
```

Comment:
I think the response is out of date too. See https://github.com/FusionAuth/fusionauth-issues/issues/2466 for more detail. Is that out of scope?